src/open_inwoner/js/components/openforms/index.js

```diff
@@ -0,0 +1,22 @@
+class ShowForm {
+  constructor(node) {
+    this.node = node
+    this.sidebar = document.querySelector('.grid__sidebar')
+    this.productDetail = document.querySelector('.product-detail')
+    this.openforms = document.querySelector('.openforms')
+    this.node.addEventListener('click', this.show.bind(this))
+  }
+
+  show(event) {
+    event.preventDefault()
+
+    this.sidebar.classList.add('grid__sidebar--hide')
+    this.productDetail.classList.add('product-detail--hide')
+    this.openforms.classList.add('openforms--show')
```

Comment:
A better way of showing/hiding things might be by setting aria-hidden as it will also remove thing for screen readers.